### DISCUSSION TITLE
[2024-07-10] 회의록

### DISCUSSION THREAD

**Discussion by ggeunn** [finalprojectSinchon/backend#5] [General]
## QR
- 시설물 등록할 때마다 QR 생성
- 시설물 삭제할 때 QR도 삭제
- QR을 찍으면 안전점검 등록 페이지로
- 안전점검 등록 페이지에 위치, 시설물 정보 , 담당자 , 날짜 자동으로 정보 불러오기
- 장비 재고 QR 발급
## 현직자 멘토링 
- 사전 질문 리스트 정리
- 노션에 기획 부분 세세하게 수정
     -  유스케이스 , 단위업무 정의서 피그마에 맞춰서 수정
     - 안전점검 , 정비, 승인 따로 빼기

**Comment by YoonJae00** [comment]
#### 멘토링질문

- JPA 활용 방법 ( JPA 를 쓰는이유가 무엇인지  )
- 웹소켓
- 프로젝트 우선순위
- 우리 기능 중에 무엇이 제일 중요한지
- AI 활용
- 이 주제가 실무에서 유용하게 쓰일지 (기능 별)
- 현재 선택한 기술스택이 우리 프로젝트에 적합한지/ 다른 추천하는 기술 스택이 있는지
- 실무에서 어떤 설계패턴으로 주로 설계하는지
- 코드 품질을 높이기 위해 어떤 방법을 사용해야 하는지
- 프로젝트 코드리뷰할때 주로 어떤 부분을 신경써야 하는지
- 프로젝트 배포할 때 주의해야 할 부분이 있는지
- 효과적인 커뮤니케이션을 위해 어떤 방법을 사용한느지(팀협업)
- 디버깅을 효율적으로 하는 팁
- 개발자로서 경력을 쌓을 때 가장 중요한 건 무엇인지
- 처음 개발자를 시작할 때 도움이 되었던 자료나 방법이 있는지
- 앞으로 유명할것 같은 기술스택이나 언어가 무엇인지
- 주목해야 할 기술트랜드나 도구가 있는지
- 우리 프로젝트에서 추가해야 할 기능이 있는지
- 보안 강화를 위해 어떤 조치를 취해야 하는지
- 데이터베이스 설계 시 주의해야 할 점
- UX 를 개선하기 위한 팁
- 코드 주석이나 API 문서화를 잘하기 위한 팁
- 첫 회사에 어떻게 들어갔는지/나이
- 어떤거 담당했는지
- 웹개발에 미래가 있는지
- 코드 짤 때 성능최적화를 생각하면서 코드를 짜는지
- 앱 할 수 있을지
- http 구조 이런거 공부하는게 좋은지
